migration-guides/ESLINT_FLAT_CONFIG.md

```diff
@@ -0,0 +1,104 @@
+# Migration to ESLint Flat Configuration
+
+[ESLint RC format](https://eslint.org/docs/latest/use/configure/configuration-files-deprecated) is deprecated in favor of [ESLint FlatConfig](https://eslint.org/blog/2022/08/new-config-system-part-2/).
+To migrate, you can follow their [migration guide](https://eslint.org/docs/latest/use/configure/migration-guide).
+
+On Otter side, an ESLint Flat Configuration is exposed by the [`@o3r/eslint-config`](https://www.npmjs.com/package/@o3r/eslint-config) package.
```

Comment:
```suggestion
On Otter side, an ESLint flat configuration is exposed by the [`@o3r/eslint-config`](https://www.npmjs.com/package/@o3r/eslint-config) package.
```

Reply:
In their doc they use the term `Flat Config` with capital F and C